pandas/compat/numpy/function.py

```diff
@@ -360,3 +360,22 @@ def validate_resampler_func(method, args, kwargs):
                 "{func}() instead".format(func=method)))
         else:
             raise TypeError("too many arguments passed in")
+
+
+def validate_minmax_axis(axis):
+    """
+    Ensure that the axis argument passed to min, max, argmin, or argmax is
+    zero or None, as otherwise it will be incorrectly ignored.
+     Parameters
```

Comment:
```suggestion
    Parameters
    ----------
    axis : int or None
    
    Raises
    ------
    ValueError
```

Reply:
This would cause a sphinx error if it were ever run through sphinx. Mostly, I just wanted to try out the "suggestion" tool.

Reply:
Yikes, yah, copy/paste screwup.  Good catch.

So far it seems like a neat feature.

Reply:
Hmm the way it got applied looks a lot like a merge mixup.